src/cdk/tree/control/flat-tree-control.spec.ts

```diff
@@ -0,0 +1,183 @@
+import {FlatTreeControl} from './flat-tree-control';
+
+describe('CdkFlatTreeControl', () => {
+  let treeControl: FlatTreeControl<TestData>;
+  let getLevel = (node: TestData) => node.level;
+  let isExpandable = (node: TestData) => node.children && node.children.length > 0;
+
+  beforeEach(() => {
+    treeControl = new FlatTreeControl<TestData>(getLevel, isExpandable);
+  });
+
+  describe('base tree control actions', () => {
+    it('should be able to expand and collapse dataNodes', () => {
+      const nodes = generateData(10, 4);
+      const node = nodes[1];
+      const sixthNode = nodes[5];
+      treeControl.dataNodes = nodes;
+
+      treeControl.expand(node);
+
+
+      expect(treeControl.isExpanded(node)).toBeTruthy('Expect second node to be expanded');
+      expect(treeControl.expansionModel.selected)
+          .toContain(node, 'Expect second node in expansionModel');
+      expect(treeControl.expansionModel.selected.length)
+          .toBe(1, 'Expect only second node in expansionModel');
+
+      treeControl.toggle(sixthNode);
+
+      expect(treeControl.isExpanded(node)).toBeTruthy('Expect second node to stay expanded');
```

Comment:
Add test to check that `isExpanded` is true for the sixthNode?

`expect(treeControl.isExpanded(sixthNode)).toBeTruthy('Expect sixth node to be expanded');`